runtime/src/rpc.rs

```diff
@@ -1209,7 +1220,7 @@ impl SecurityPallet for InterBtcParachain {
     }
 
     /// Return any `ErrorCode`s set in the security module.
-    async fn get_error_codes(&self) -> Result<BTreeSet<ErrorCode>, Error> {
+    async fn get_error_codes(&self) -> Result<Vec<ErrorCode>, Error> {
```

Comment:
Not sure why this changed but it doesn't matter since we will anyway remove these error codes in the next release